message/message.pb.go

```diff
@@ -355,8 +355,9 @@ func (m *Address) GetAddress() string {
 }
 
 type Registration struct {
-	Amount uint64      `protobuf:"varint,1,opt,name=amount" json:"amount,omitempty"`
-	Type   ShuffleType `protobuf:"varint,2,opt,name=type,enum=ShuffleType" json:"type,omitempty"`
+	Amount  uint64      `protobuf:"varint,1,opt,name=amount" json:"amount,omitempty"`
+	Type    ShuffleType `protobuf:"varint,2,opt,name=type,enum=ShuffleType" json:"type,omitempty"`
+	Version uint64      `protobuf:"varint,3,opt,name=version" json:"version,omitempty"`
```

Comment:
Sorry if it wasn't cashshuffle, but I remember somebody talking about missing message fields causing a problem. Is `omitempty` intentional / not a problem here?

Reply:
This is fine for this use case. The default value is always generated client side if omitted in the protobuf stream.

Reply:
Thanks for explaining. I should have known that :) Whatever I remember must have been somewhere else.